pkg/infra/tracing/opentelemetry_tracing.go

```diff
@@ -115,20 +128,46 @@ func (ots *Opentelemetry) parseSettingsOpentelemetry() error {
 	return nil
 }
 
+func splitCustomAttribs(s string) ([]attribute.KeyValue, error) {
+	res := []attribute.KeyValue{}
+
+	attribs := strings.Split(s, ",")
+	for _, v := range attribs {
+		parts := strings.SplitN(v, ":", 2)
+		if len(parts) > 1 {
+			res = append(res, attribute.String(parts[0], parts[1]))
+		} else if v != "" {
+			return nil, fmt.Errorf("custom attribute malformed - must be in 'key:value' form: %q", v)
+		}
+	}
+
+	return res, nil
+}
+
 func (ots *Opentelemetry) initJaegerTracerProvider() (*tracesdk.TracerProvider, error) {
 	// Create the Jaeger exporter
 	exp, err := jaeger.New(jaeger.WithCollectorEndpoint(jaeger.WithEndpoint(ots.address)))
 	if err != nil {
 		return nil, err
 	}
 
-	tp := tracesdk.NewTracerProvider(
-		tracesdk.WithBatcher(exp),
-		tracesdk.WithResource(resource.NewWithAttributes(
-			semconv.SchemaURL,
+	res, err := resource.New(
+		context.Background(),
+		resource.WithAttributes(
+			// TODO: why are these attributes different from ones added to the
+			// OTLP provider?
 			semconv.ServiceNameKey.String("grafana"),
```

Comment:
I agree with the comment, tracing needs some more attention, I was not able to prioritize it so far